ckanext/collaborators/templates/collaborator/collaborator_new.html

```diff
@@ -0,0 +1,73 @@
+{% extends "package/edit_base.html" %}
```

Comment:
This page doesn't render properly for me (sorry if it's still WIP): selects look wrong and there's no Submit button:
![Screenshot_2019-05-31 Add Collaborator - Datasets - CKAN](https://user-images.githubusercontent.com/200230/58696930-e4e52180-8398-11e9-8d71-523f87287423.png)

Reply:
When developing using okfn/docker-ckan it works ok:
![add collaborator 2](https://user-images.githubusercontent.com/6672339/58716415-5a53f080-839f-11e9-9c5b-cee445661cec.png)

I could reproduce this error in another project and it seems a css problem because the button is rendered and it works but there is a problem on how it is displayed:

![button add collaborator](https://user-images.githubusercontent.com/6672339/58716514-8e2f1600-839f-11e9-8923-0c2d786bc301.png)